README.md

```diff
@@ -32,44 +32,52 @@ optional arguments:
 The `update` sub-command modifies ontology version and dependency information
 ```
 $ onto_tool update -h
-usage: onto_tool update [-h] [-b] [-v SET_VERSION]
-                        [-i [VERSION_INFO]] [-d DEPENDENCY VERSION]
+usage: onto_tool update [-h] [-f {xml,turtle,nt} | -i] [-o OUTPUT] [-b]
+                        [-v SET_VERSION] [--version-info [VERSION_INFO]]
+                        [-d DEPENDENCY VERSION]
                         [ontology [ontology ...]]
 
 positional arguments:
-  ontology              Ontology file or directory containing OWL files.
+  ontology              Ontology file or directory containing OWL files
 
 optional arguments:
   -h, --help            show this help message and exit
+  -f {xml,turtle,nt}, --format {xml,turtle,nt}
+                        Output format
+  -i, --in-place        Overwrite each input file with update, preserving
+                        format
+  -o OUTPUT, --output OUTPUT
+                        Path to output file. Will be ignored if --in-place is
```

Comment:
Doesn't argparse issue an error if more than one arg of a mutually exclusive group is provided? In that case the comment should be something like 'Not allowed if --in-place is specified', and similarly for the other mutually exclusive args. Unless you are doing error-checking first and ignoring one of them.

Reply:
The `output_format_group` mutually exclusive group only includes `--format` and `--in-place`, since the latter preserves format. `--output` is compatible with `--format`, so can't go in the same group (it's directly in the `update_parser`), and is just ignored when `--in-place` is provided, as documented.